src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -335,8 +335,8 @@ internal virtual bool AddIfNotPresent(T item)
             Node grandParent = null;
             Node greatGrandParent = null;
 
-            // even if we don't actually add to the set, we may be altering its structure (by doing rotations
-            // and such). so update version to disable any enumerators/subsets working on it
+            // Even if we don't actually add to the set, we may be altering its structure (by doing rotations and such).
```

Comment:
In the future, it'd be helpful to avoid doing lots of additional subsequent changes like this after the PR has already been reviewed.  It makes reviewing more difficult and time consuming.  Thanks.